src/core/Akka.API.Tests/CoreAPISpec.ApproveStreams.approved.txt

```diff
@@ -4400,21 +4402,17 @@ namespace Akka.Streams.Stage
         protected PushStage() { }
         public virtual Akka.Streams.Stage.ISyncDirective OnPull(Akka.Streams.Stage.IContext<TOut> context) { }
     }
-    public sealed class StageActorRef : Akka.Actor.MinimalActorRef
+    public sealed class StageActor
```

Comment:
Should we move this to 1.4 for the breaking changes?

Reply:
Depends on how do we threat a `[ApiMayChange]` attribute - only methods with that API have been changed. We can put it to v1.4 - from my perspective it's necessary for purposes of StreamRefs, which probably will land somewhere in v1.4 as well.

Reply:
Except for the `StageActorRef` property itself